docs/add-ons/tutorial.md

```diff
@@ -117,7 +117,7 @@ RUN apk add --no-cache python3
 WORKDIR /data
 ```
 
-Add "ports" to `config.json`. This will make TCP on port 8000 inside the container available on the host on port 8000.
+Add "ingress" options to `config.json`. Ingress allow users to access the add-on web interface via the Home Assistant UI. Authentication is handled by Home Assistant, so neither the user nor the add-on developer will need to care about the security or port forwarding. 
```

Comment:
Small tweak:

```suggestion
Add "ingress" options to `config.json`. Ingress allows users to access the add-on web interface via the Home Assistant UI. Authentication is handled by Home Assistant, so neither the user nor the add-on developer will need to care about the security or port forwarding. 
```

I do doubt this change a bit. Because "nor the add-on developer will need to care about the security" is not entirely true. There are actually quite a few things an add-on developer should take care of, for example, if the service is available on the port directly, it bypasses HA and thus still needs solving.

My worry with this change: Giving a false sense of security.

Reply:
I've taken that text from https://developers.home-assistant.io/docs/add-ons/presentation. Might need rewording there as well.